addons/resource.language.en_gb/resources/strings.po

```diff
@@ -13861,7 +13861,19 @@ msgctxt "#21345"
 msgid "Uncategorized"
 msgstr ""
 
-#empty strings from id 21346 to 21357
+#. Used to give heading list entry for an file extension description, e.g. to show as 'Extension: ".mp3"'
```

Comment:
I'll get grammar improvements for the string descriptions in a bit. The comments are mostly fine, do you still want recommendations there?